extensions/amp-story/1.0/amp-story-consent.js

```diff
@@ -280,8 +284,22 @@ export class AmpStoryConsent extends AMP.BaseElement {
           .classList.toggle('i-amphtml-story-consent-fullbleed', isFullBleed);
     };
 
-    this.element.getResources()
-        .measureMutateElement(this.storyConsentEl_, measurer, mutator);
+    this.measureMutateElement(measurer, mutator, this.storyConsentEl_);
+  }
+
+  /**
+   * Reacts to RTL state updates and triggers the UI for RTL.
+   * @param {boolean} rtlState
+   * @private
+   */
+  onRtlStateUpdate_(rtlState) {
+    const mutator = () => {
+      rtlState ?
+        this.storyConsentEl_.setAttribute('dir', 'rtl') :
+        this.storyConsentEl_.removeAttribute('dir');
+    };
+
+    this.mutateElement(mutator, this.storyConsentEl_);
```

Comment:
Nit: the element is optional, you don't need to pass it as an argument.
Up to you, please pick the syntax you like the most, but you could also write:

````
this.mutateElement(() => {
  rtlState ?
    this.storyConsentEl_.setAttribute('dir', 'rtl') :
    this.storyConsentEl_.removeAttribute('dir');
});
````

Reply:
I decided to stick with what we were doing with this file and pass `this.storyConsentEl_` instead of `this.element` in the `mutate` since they seem to be different and didn't have much context on their differences. If you think it's ok, I can make the change so that we pass `this.element` instead :)